guide_drafts/model-derives.md

```diff
@@ -0,0 +1,935 @@
+           `diesel_demo::models::Post : diesel::associations::HasTable`
+           `diesel_demo::models::Post : diesel::associations::BelongsTo<_>`
+           `diesel_demo::models::Post : diesel::associations::BelongsTo<diesel_demo::models::User>`
+           `diesel_demo::models::Post : diesel::associations::HasTable`
+           `diesel_demo::models::Post : diesel::associations::BelongsTo<diesel_demo::models::User>`
+           `diesel_demo::models::Post : diesel::associations::BelongsTo<diesel_demo::models::User>`
+           `&diesel_demo::models::Post : diesel::BelongingToDsl<&[_]>`
+           `&diesel_demo::models::Post : diesel::associations::HasTable`
+           `&diesel_demo::models::Post : diesel::associations::BelongsTo<_>`
+           `&diesel_demo::models::Post : diesel::associations::BelongsTo<diesel_demo::models::User>`
+           `diesel_demo::models::Post : diesel::associations::HasTable`
+           `diesel_demo::models::Post : diesel::associations::HasTable`
+           `&diesel_demo::models::Post : diesel::associations::BelongsTo<diesel_demo::models::User>`
+           `&diesel_demo::models::Post : diesel::associations::BelongsTo<diesel_demo::models::User>`
+           `&diesel_demo::models::Post : diesel::associations::HasTable`
+           `&diesel_demo::models::Post : diesel::associations::BelongsTo<_>`
+           `&diesel_demo::models::Post : diesel::associations::BelongsTo<diesel_demo::models::User>`
+           `diesel_demo::models::Post : diesel::associations::HasTable`
+           `diesel_demo::models::Post : diesel::associations::HasTable`
+           `&diesel_demo::models::Post : diesel::associations::BelongsTo<diesel_demo::models::User>`
+           `&diesel_demo::models::Post : diesel::associations::BelongsTo<diesel_demo::models::User>`
+           `&mut diesel_demo::models::Post : diesel::BelongingToDsl<&[_]>`
+           `&mut diesel_demo::models::Post : diesel::associations::HasTable`
+           `&mut diesel_demo::models::Post : diesel::associations::BelongsTo<_>`
+           `&mut diesel_demo::models::Post : diesel::associations::BelongsTo<diesel_demo::models::User>`
+           `&mut diesel_demo::models::Post : diesel::associations::HasTable`
+           `&mut diesel_demo::models::Post : diesel::associations::BelongsTo<diesel_demo::models::User>`
+           `&mut diesel_demo::models::Post : diesel::associations::BelongsTo<diesel_demo::models::User>`
+           `&mut diesel_demo::models::Post : diesel::associations::HasTable`
+           `&mut diesel_demo::models::Post : diesel::associations::BelongsTo<_>`
+           `&mut diesel_demo::models::Post : diesel::associations::BelongsTo<diesel_demo::models::User>`
+           `&mut diesel_demo::models::Post : diesel::associations::HasTable`
+           `&mut diesel_demo::models::Post : diesel::associations::BelongsTo<diesel_demo::models::User>`
+           `&mut diesel_demo::models::Post : diesel::associations::BelongsTo<diesel_demo::models::User>`
```

Comment:
Yeah you can probably replace the last 37 lines with ellipses

Reply:
😁